web/ui/src/pages/PageComponentList.tsx

```diff
@@ -1,15 +1,44 @@
 import { faCubes } from '@fortawesome/free-solid-svg-icons';
 
 import ComponentList from '../features/component/ComponentList';
+import { SortOrder } from '../features/component/types';
 import Page from '../features/layout/Page';
 import { useComponentInfo } from '../hooks/componentInfo';
 
+const fieldMappings: { [key: string]: (comp: any) => string | undefined } = {
+  Health: (comp) => comp.health?.state?.toString(),
+  ID: (comp) => comp.localID,
+  // Add new fields if needed here.
+};
+
+function getSortValue(component: any, field: string): string | undefined {
```

Comment:
nit: we actually should replace any (+ the one ligne 8) with Component. This was my mistake in the code I suggested and was caught by the linter: " Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any"

Reply:
Fixed in latest commit